Rebranding of secplug to Scancloud APIs

Ajaatsecplugs opened the pull request:
Rebranding of secplug to Scancloud APIs


---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

Amjed-Ayoub commented:
Hi @Ajaatsecplugs, thank you so much for submitting your pull request. This is a new verified connector. Like all verified publisher connectors, we ask that you register your connector for certification by filling out our [registration form](https://aka.ms/ConnectorRegistration).
Please let me know if you have any questions.
Thank you very much again for working with us.

Amjed-Ayoub commented:
Apologies for this mistake. You are correct. No need for registration again :-) we will review this pull request soon.
Thank you very much.

v-EgorKozhadei commented on certified-connectors/secplugs/apiDefinition.swagger.json:
```diff
@@ -1,12 +1,12 @@
   "info": {
-    "title": "Secplugs Scan",
-    "description": "Secplugs Scan for scanning Emails content, files contents and Urls, Secplugs scan is a platform which creates hybrid security platform using various security provider",
+    "title": "Scancloud APIs",
```
@Ajaatsecplugs please remove the workdAPI from the title and from the description, it's not allowed to use it. You can check the documentation [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info) and [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title) for more information.

v-EgorKozhadei replied:
Title update is not allowed for deployed connectors, it's a breaking change. Can you please revert the title to Secplugs Scan? If you want a new name you'll have to submit a new PR following [these guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission).

v-EgorKozhadei commented on certified-connectors/secplugs/apiDefinition.swagger.json:
```diff
@@ -1,12 +1,12 @@
-      "name": "Secplugs",
-      "url": "https://secplugs.com/",
+      "name": "Scancloud",
+      "url": "Website	https://scancloud.io/",
```
@Ajaatsecplugs the URL property should contain only contact URL, please remove the word Website and whitespaces from the string.

v-EgorKozhadei reviewed (commented):
Hi @Ajaatsecplugs! Please look at the commends after the review, thank you!

v-EgorKozhadei commented on certified-connectors/secplugs/readme.md:
```diff
@@ -1,20 +1,20 @@
+You would need to ensure that you have an account in secplugs and using https://scancloud.io/index.php/providers an power automate profile is created.
 After that is completed, you can create and test the secplugs connector.
 
 ### Deploying the sample
```
@Ajaatsecplugs please format the readme document according to [the readme.md template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md).

Ajaatsecplugs replied:
Fixed

Ajaatsecplugs commented:
We would like to update the Title for the deployed connector - As per your
suggestion, we have followed the guidelines
https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission
and
raised a new PR with the new title.

https://github.com/microsoft/PowerPlatformConnectors/pull/1697

Thanks
Shankar

On Mon, 23 May 2022 at 21:44, Amjed Ayoub ***@***.***> wrote:

> Apologies for this mistake. You are correct. No need for registration
> again :-) we will review this pull request soon.
> Thank you very much.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/microsoft/PowerPlatformConnectors/pull/1668#issuecomment-1134873601>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOYRTYILXKSBSZOMDNCTO63VLOVGVANCNFSM5WTJFCMA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Ajaatsecplugs commented:
We have completed the registration and followed the guidelines - Please
find the PR:
https://github.com/microsoft/PowerPlatformConnectors/pull/1697/files

Please review and confirm.

thanks
Shankar

On Tue, 31 May 2022 at 11:13, Shankar Jayaraman ***@***.***>
wrote:

> We would like to update the Title for the deployed connector - As per your
> suggestion, we have followed the guidelines
> https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission and
> raised a new PR with the new title.
>
> https://github.com/microsoft/PowerPlatformConnectors/pull/1697
>
> Thanks
> Shankar
>
> On Mon, 23 May 2022 at 21:44, Amjed Ayoub ***@***.***>
> wrote:
>
>> Apologies for this mistake. You are correct. No need for registration
>> again :-) we will review this pull request soon.
>> Thank you very much.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/microsoft/PowerPlatformConnectors/pull/1668#issuecomment-1134873601>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AOYRTYILXKSBSZOMDNCTO63VLOVGVANCNFSM5WTJFCMA>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

Ajaatsecplugs commented:
Could you please acknowledge and let us know the ETA to review and merge
the PR?

thanks
Shankar

On Sun, 5 Jun 2022 at 16:24, Shankar Jayaraman ***@***.***> wrote:

> We have completed the registration and followed the guidelines - Please
> find the PR:
> https://github.com/microsoft/PowerPlatformConnectors/pull/1697/files
>
> Please review and confirm.
>
> thanks
> Shankar
>
> On Tue, 31 May 2022 at 11:13, Shankar Jayaraman ***@***.***>
> wrote:
>
>> We would like to update the Title for the deployed connector - As per
>> your suggestion, we have followed the guidelines
>> https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission and
>> raised a new PR with the new title.
>>
>> https://github.com/microsoft/PowerPlatformConnectors/pull/1697
>>
>> Thanks
>> Shankar
>>
>> On Mon, 23 May 2022 at 21:44, Amjed Ayoub ***@***.***>
>> wrote:
>>
>>> Apologies for this mistake. You are correct. No need for registration
>>> again :-) we will review this pull request soon.
>>> Thank you very much.
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/microsoft/PowerPlatformConnectors/pull/1668#issuecomment-1134873601>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AOYRTYILXKSBSZOMDNCTO63VLOVGVANCNFSM5WTJFCMA>
>>> .
>>> You are receiving this because you were mentioned.Message ID:
>>> ***@***.***>
>>>
>>

v-EgorKozhadei commented:
Hi @Ajaatsecplugs!

We will review this PR within 1-2 weeks.

It turned out that we can't have two connectors with the same functionality under different names.
I'm sorry that we asked you to open a new PR because of the new name.
We return to reviewing this PR, [the new one](https://github.com/microsoft/PowerPlatformConnectors/pull/1697) will be closed soon.

Can you please apply all the file changes that you have made in [the new PR](https://github.com/microsoft/PowerPlatformConnectors/pull/1697) to this PR?

Thank you in advance! And sorry for inconveniences!

v-EgorKozhadei commented on certified-connectors/Scancloud/apiDefinition.swagger.json:
```diff
@@ -1,12 +1,12 @@
-    "title": "Secplugs Scan",
-    "description": "Secplugs Scan for scanning Emails content, files contents and Urls, Secplugs scan is a platform which creates hybrid security platform using various security provider",
+    "title": "Scancloud",
+    "description": "Scancloud for scanning Emails content, files contents and Urls, Scancloud is a platform which creates hybrid security platform using various security provider",
```
@Ajaatsecplugs can you please describe concisely the main purpose and value offered by your connector in the description? You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

v-EgorKozhadei commented on certified-connectors/Scancloud/apiDefinition.swagger.json:
```diff
@@ -338,7 +338,7 @@
         },
-        "summary": "Scan Emails using SecPlugs Scan",
+        "summary": "Scan Emails using Scancloud",
         "operationId": "emailscan",
```
@Ajaatsecplugs please [use pascal casing for operationId](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers) properties.

v-EgorKozhadei commented on certified-connectors/Scancloud/apiDefinition.swagger.json:
```diff
@@ -154,8 +154,8 @@
-            "description": "Secplugs API Key",
-            "x-ms-summary": "Secplugs API Key"
+            "description": "Scancloud API Key",
+            "x-ms-summary": "Scancloud API Key"
```
@Ajaatsecplugs please don't copy paste summaries and descriptions, they should differ. Can you please make a bit more detailed descriptions? You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

v-EgorKozhadei commented on certified-connectors/Scancloud/apiDefinition.swagger.json:
```diff
@@ -347,8 +347,8 @@
             "type": "string",
             "default": "",
```
@Ajaatsecplugs empty properties are not allowed. You can remove it until required or define a value. Please refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).

v-EgorKozhadei commented on certified-connectors/Scancloud/readme.md:
```diff
@@ -0,0 +1,30 @@
+
+## Scancloud Connector
```
@Ajaatsecplugs please remove the word Connector from the title.

v-EgorKozhadei commented on certified-connectors/Scancloud/readme.md:
```diff
@@ -0,0 +1,30 @@
+
+## Scancloud Connector
+Scancloud provides a powerful platform for pluggable security using Scancloud plugins.
```
@Ajaatsecplugs please provide some information about the main purpose and value offered by your connector.

Ajaatsecplugs replied:
Added

v-EgorKozhadei commented on certified-connectors/Scancloud/readme.md:
```diff
@@ -0,0 +1,30 @@
+## Known Issues and Limitations
+N/A
+
+### Deploying the sample
```
@Ajaatsecplugs please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1). All of the required headings should be present.

Ajaatsecplugs replied:
Added

v-EgorKozhadei reviewed (commented):
Hi @Ajaatsecplugs! Please check comments after the review, thank you!

v-EgorKozhadei commented:
Hi @Ajaatsecplugs! Can you please also leave the connector folder name as it is? So the files will be updated, but the connector folder name will stay the same.

Ajaatsecplugs commented:
Thank you for the review comments - Fixed all the review comments and
raised the PR again..

Thanks
Shankar

On Sat, 11 Jun 2022 at 00:19, v-EgorKozhadei ***@***.***>
wrote:

> Hi @Ajaatsecplugs <https://github.com/Ajaatsecplugs>! Can you please also
> leave the connector folder name as it is? So the files will be updated, but
> the connector folder name will stay the same.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/microsoft/PowerPlatformConnectors/pull/1668#issuecomment-1152641441>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOYRTYLTG2FQZ7WGYH35N4LVOOE3RANCNFSM5WTJFCMA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

v-EgorKozhadei commented:
Hi @Ajaatsecplugs!

Your PR is approved and merged. Please submit an update in ISV Studio by using your latest commit ID.

Please let me know if you have any questions.

Ajaatsecplugs commented:
Powerplatform team,

I understand that scancloud connectors are now live in power
automate connectors site.. I understand that you wanted to treat
Scancloud.io connectors as new connectors not merging with existing
secplugs connectors.. There seems to be some issue here..

1. Secplugs and scancloud connector documentation got mixed up..
https://docs.microsoft.com/en-us/connectors/secplugscan/
This is quite confusing - We would like to stick on to the latest
documentation on scancloud connectors - you can remove the secplugs
documentation all together if you want..

2. Scancloud connectors - says "scan newly added files using Secplugs"..
https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/
As we mentioned earlier we want to replace secplugs completely with
Scancloud.io - this is a rebranding exercise as mentioned earlier.. Please
remove secplugs and replace them with Scancloud..  It should not get mixed
up..

Please let us know if you have any questions.

thanks
Shankar

On Sat, 25 Jun 2022 at 02:41, v-EgorKozhadei ***@***.***>
wrote:

> Hi @Ajaatsecplugs <https://github.com/Ajaatsecplugs>!
>
> Your PR is approved and merged. Please submit an update in ISV Studio by
> using your latest commit ID.
>
> Please let me know if you have any questions.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/microsoft/PowerPlatformConnectors/pull/1668#issuecomment-1165943366>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOYRTYIFK4ZOTIML7DITG2TVQYP7ZANCNFSM5WTJFCMA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Ajaatsecplugs commented:
Hello Shankar,

Thank you so much for reaching out.

You new update deployment did not start yet 😊. Hopefully, our engineers will prepare your update for Friday’s deployment train. Typically, it takes up to 3 to 4 weeks to complete.

Please let me know if you have any questions.

Thank you so much for your patience and understanding.

Best regards,
Amjed Ayoub

From: Shankar Jayaraman ***@***.***>
Sent: Sunday, August 7, 2022 9:07 PM
To: microsoft/PowerPlatformConnectors ***@***.***>
Cc: microsoft/PowerPlatformConnectors ***@***.***>; Kartik Rao Polepalli ***@***.***>; Connector Partner Mgmt Team ***@***.***>; Srikanth Yenagandhula (MINDTREE LIMITED) ***@***.***>
Subject: [EXTERNAL] Re: [microsoft/PowerPlatformConnectors] Rebranding of secplug to Scancloud APIs (PR #1668)

Powerplatform team,

I understand that scancloud connectors are now live in power automate connectors site.. I understand that you wanted to treat Scancloud.io connectors as new connectors not merging with existing secplugs connectors.. There seems to be some issue here..

1. Secplugs and scancloud connector documentation got mixed up..
https://docs.microsoft.com/en-us/connectors/secplugscan/<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fconnectors%2Fsecplugscan%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=mjws3CF0KbU%2F%2BjUTP2wf8yXdcBLJga0K8xl1kstqrzo%3D&reserved=0>
This is quite confusing - We would like to stick on to the latest documentation on scancloud connectors - you can remove the secplugs documentation all together if you want..

2. Scancloud connectors - says "scan newly added files using Secplugs"..
https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fconnectors%2Fdetails%2Fshared_secplugscan%2Fscancloud%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=xRUxiLogvbWV7rzoUKlQrc6rh9c%2FClzoPVLsIp8njso%3D&reserved=0>
As we mentioned earlier we want to replace secplugs completely with Scancloud.io - this is a rebranding exercise as mentioned earlier.. Please remove secplugs and replace them with Scancloud..  It should not get mixed up..

Please let us know if you have any questions.

thanks
Shankar

On Sat, 25 Jun 2022 at 02:41, v-EgorKozhadei ***@***.******@***.***>> wrote:

Hi @Ajaatsecplugs<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAjaatsecplugs&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=GumyA%2B1yrKgYkmw5sEtoePy6OJfc5zQRQTSXfAujzG8%3D&reserved=0>!

Your PR is approved and merged. Please submit an update in ISV Studio by using your latest commit ID.

Please let me know if you have any questions.

—
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FPowerPlatformConnectors%2Fpull%2F1668%23issuecomment-1165943366&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=UN4nazsbYkTpMAICizm%2F4IfcNPh0aum%2BlYQcyHyceSc%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOYRTYIFK4ZOTIML7DITG2TVQYP7ZANCNFSM5WTJFCMA&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=K0Ix10CJ5O6KzCyZT8aMHUqKaQoCjEPV4%2FD%2FBIc3g4U%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

Ajaatsecplugs commented:
Okay thanks for the update..  The below message we see in the activity
center..  Hope this is the correct status..

"
Dear Partner,

Your connector is now onboarded to the current deployment schedule. You can
expect another status update in 3-5 business days.

Best Regards, Srikanth Y.
"
Thanks
Shankar


On Tue, 9 Aug 2022 at 05:29, Amjed Ayoub (TEKsystems, Inc.) <
***@***.***> wrote:

> Hello Shankar,
>
>
>
> Thank you so much for reaching out.
>
>
>
> You new update deployment did not start yet 😊. Hopefully, our engineers
> will prepare your update for Friday’s deployment train. Typically, it takes
> up to 3 to 4 weeks to complete.
>
>
>
> Please let me know if you have any questions.
>
>
>
> Thank you so much for your patience and understanding.
>
>
>
> Best regards,
>
> Amjed Ayoub
>
>
>
> *From:* Shankar Jayaraman ***@***.***>
> *Sent:* Sunday, August 7, 2022 9:07 PM
> *To:* microsoft/PowerPlatformConnectors <
> ***@***.***>
> *Cc:* microsoft/PowerPlatformConnectors <
> ***@***.***>; Kartik Rao Polepalli <
> ***@***.***>; Connector Partner Mgmt Team <
> ***@***.***>; Srikanth Yenagandhula
> (MINDTREE LIMITED) ***@***.***>
> *Subject:* [EXTERNAL] Re: [microsoft/PowerPlatformConnectors] Rebranding
> of secplug to Scancloud APIs (PR #1668)
>
>
>
> Powerplatform team,
>
>
>
> I understand that scancloud connectors are now live in power
> automate connectors site.. I understand that you wanted to treat
> Scancloud.io connectors as new connectors not merging with existing
> secplugs connectors.. There seems to be some issue here..
>
>
>
> 1. Secplugs and scancloud connector documentation got mixed up..
>
> https://docs.microsoft.com/en-us/connectors/secplugscan/
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fconnectors%2Fsecplugscan%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=mjws3CF0KbU%2F%2BjUTP2wf8yXdcBLJga0K8xl1kstqrzo%3D&reserved=0>
>
> This is quite confusing - We would like to stick on to the latest
> documentation on scancloud connectors - you can remove the secplugs
> documentation all together if you want..
>
>
>
> 2. Scancloud connectors - says "scan newly added files using Secplugs"..
>
>
> https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fconnectors%2Fdetails%2Fshared_secplugscan%2Fscancloud%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=xRUxiLogvbWV7rzoUKlQrc6rh9c%2FClzoPVLsIp8njso%3D&reserved=0>
>
> As we mentioned earlier we want to replace secplugs completely with
> Scancloud.io - this is a rebranding exercise as mentioned earlier.. Please
> remove secplugs and replace them with Scancloud..  It should not get mixed
> up..
>
>
>
> Please let us know if you have any questions.
>
>
>
> thanks
>
> Shankar
>
>
>
> On Sat, 25 Jun 2022 at 02:41, v-EgorKozhadei ***@***.***>
> wrote:
>
> Hi @Ajaatsecplugs
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAjaatsecplugs&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=GumyA%2B1yrKgYkmw5sEtoePy6OJfc5zQRQTSXfAujzG8%3D&reserved=0>
> !
>
> Your PR is approved and merged. Please submit an update in ISV Studio by
> using your latest commit ID.
>
> Please let me know if you have any questions.
>
> —
> Reply to this email directly, view it on GitHub
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FPowerPlatformConnectors%2Fpull%2F1668%23issuecomment-1165943366&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=UN4nazsbYkTpMAICizm%2F4IfcNPh0aum%2BlYQcyHyceSc%3D&reserved=0>,
> or unsubscribe
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOYRTYIFK4ZOTIML7DITG2TVQYP7ZANCNFSM5WTJFCMA&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=K0Ix10CJ5O6KzCyZT8aMHUqKaQoCjEPV4%2FD%2FBIc3g4U%3D&reserved=0>
> .
> You are receiving this because you were mentioned.[image: Image removed
> by sender.]Message ID: <microsoft/PowerPlatformConnectors/pull/1668/
> ***@***.***>
>
>

Ajaatsecplugs commented:
Hi Team - There are some issues with scancloud connector preview  - I can
see our scancloud rebranded connector updated as a preview
https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/
 - Based on our earlier conversation, you have mentioned that you will be
uploading this as a brand new scancloud connector, not an update to the
existing secplugs connector. Also, we have the following concerns. Request
to please address the same..

1. Documentation link still shows secplugsscan
https://docs.microsoft.com/en-us/connectors/secplugscan/

2. In the connector page
https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/
It is showing as secplugs connector - In the URL, connector name etc.,

3. In the documentation "Getting started with your connector" section,
there are some broken links and images, we have provided in our
documentation.. It was there in our earlier secplugs documentation page and
we want to remove that. It looks bad and needs to be removed.. We want to
keep only one line in that section which is " Please follow ScanCloud
Connector documentation here
<https://scancloud.io/docs?plugin=plugin-secplugs-plugins-power-automate>".

Please remove the below section:
"
Creating a connection | DEFAULT | | |
|---------------------------------------------|---------------------|---------------------------------|
| Name | Type | Description | | API Key (from https://Scancloud.com/
<https://scancloud.com/>) | securestring | The Api Key for this API |

Clean report Sample (For File): [image: enter image description here]

Malicious report Sample (For File): [image: enter image description here]
"
Thanks
Shankar


On Tue, 9 Aug 2022 at 17:37, Shankar Jayaraman ***@***.***> wrote:

> Okay thanks for the update..  The below message we see in the activity
> center..  Hope this is the correct status..
>
> "
> Dear Partner,
>
> Your connector is now onboarded to the current deployment schedule. You
> can expect another status update in 3-5 business days.
>
> Best Regards, Srikanth Y.
> "
> Thanks
> Shankar
>
>
> On Tue, 9 Aug 2022 at 05:29, Amjed Ayoub (TEKsystems, Inc.) <
> ***@***.***> wrote:
>
>> Hello Shankar,
>>
>>
>>
>> Thank you so much for reaching out.
>>
>>
>>
>> You new update deployment did not start yet 😊. Hopefully, our engineers
>> will prepare your update for Friday’s deployment train. Typically, it takes
>> up to 3 to 4 weeks to complete.
>>
>>
>>
>> Please let me know if you have any questions.
>>
>>
>>
>> Thank you so much for your patience and understanding.
>>
>>
>>
>> Best regards,
>>
>> Amjed Ayoub
>>
>>
>>
>> *From:* Shankar Jayaraman ***@***.***>
>> *Sent:* Sunday, August 7, 2022 9:07 PM
>> *To:* microsoft/PowerPlatformConnectors <
>> ***@***.***>
>> *Cc:* microsoft/PowerPlatformConnectors <
>> ***@***.***>; Kartik Rao Polepalli <
>> ***@***.***>; Connector Partner Mgmt Team <
>> ***@***.***>; Srikanth Yenagandhula
>> (MINDTREE LIMITED) ***@***.***>
>> *Subject:* [EXTERNAL] Re: [microsoft/PowerPlatformConnectors] Rebranding
>> of secplug to Scancloud APIs (PR #1668)
>>
>>
>>
>> Powerplatform team,
>>
>>
>>
>> I understand that scancloud connectors are now live in power
>> automate connectors site.. I understand that you wanted to treat
>> Scancloud.io connectors as new connectors not merging with existing
>> secplugs connectors.. There seems to be some issue here..
>>
>>
>>
>> 1. Secplugs and scancloud connector documentation got mixed up..
>>
>> https://docs.microsoft.com/en-us/connectors/secplugscan/
>> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fconnectors%2Fsecplugscan%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=mjws3CF0KbU%2F%2BjUTP2wf8yXdcBLJga0K8xl1kstqrzo%3D&reserved=0>
>>
>> This is quite confusing - We would like to stick on to the latest
>> documentation on scancloud connectors - you can remove the secplugs
>> documentation all together if you want..
>>
>>
>>
>> 2. Scancloud connectors - says "scan newly added files using Secplugs"..
>>
>>
>> https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/
>> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fconnectors%2Fdetails%2Fshared_secplugscan%2Fscancloud%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=xRUxiLogvbWV7rzoUKlQrc6rh9c%2FClzoPVLsIp8njso%3D&reserved=0>
>>
>> As we mentioned earlier we want to replace secplugs completely with
>> Scancloud.io - this is a rebranding exercise as mentioned earlier.. Please
>> remove secplugs and replace them with Scancloud..  It should not get mixed
>> up..
>>
>>
>>
>> Please let us know if you have any questions.
>>
>>
>>
>> thanks
>>
>> Shankar
>>
>>
>>
>> On Sat, 25 Jun 2022 at 02:41, v-EgorKozhadei ***@***.***>
>> wrote:
>>
>> Hi @Ajaatsecplugs
>> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAjaatsecplugs&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=GumyA%2B1yrKgYkmw5sEtoePy6OJfc5zQRQTSXfAujzG8%3D&reserved=0>
>> !
>>
>> Your PR is approved and merged. Please submit an update in ISV Studio by
>> using your latest commit ID.
>>
>> Please let me know if you have any questions.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FPowerPlatformConnectors%2Fpull%2F1668%23issuecomment-1165943366&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=UN4nazsbYkTpMAICizm%2F4IfcNPh0aum%2BlYQcyHyceSc%3D&reserved=0>,
>> or unsubscribe
>> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOYRTYIFK4ZOTIML7DITG2TVQYP7ZANCNFSM5WTJFCMA&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=K0Ix10CJ5O6KzCyZT8aMHUqKaQoCjEPV4%2FD%2FBIc3g4U%3D&reserved=0>
>> .
>> You are receiving this because you were mentioned.[image: Image removed
>> by sender.]Message ID: <microsoft/PowerPlatformConnectors/pull/1668/
>> ***@***.***>
>>
>>

Ajaatsecplugs commented:
Hello Shankar,

Thank you so much for reaching out.

Please see my answers inline below.

Please let me know if you have any questions.

Thank you so much again for your hard work with us.

Best regards,
Amjed Ayoub




From: Shankar Jayaraman ***@***.***>
Sent: Monday, August 22, 2022 6:14 AM
To: Amjed Ayoub (TEKsystems, Inc.) ***@***.***>
Cc: microsoft/PowerPlatformConnectors ***@***.***>; microsoft/PowerPlatformConnectors ***@***.***>; Kartik Rao Polepalli ***@***.***>; Connector Partner Mgmt Team ***@***.***>; Srikanth Yenagandhula (MINDTREE LIMITED) ***@***.***>
Subject: Re: [EXTERNAL] Re: [microsoft/PowerPlatformConnectors] Rebranding of secplug to Scancloud APIs (PR #1668)

Hi Team - There are some issues with scancloud connector preview  - I can see our scancloud rebranded connector updated as a preview https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fconnectors%2Fdetails%2Fshared_secplugscan%2Fscancloud%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lT0JUW69cDlpa2VUu%2BaE8BnIaYtMgf9UQDVEJDn0U7g%3D&reserved=0>   - Based on our earlier conversation, you have mentioned that you will be uploading this as a brand new scancloud connector, not an update to the existing secplugs connector. Also, we have the following concerns. Request to please address the same..
All our new connectors will be published with this “Preview” tag. This does not mean that your connector is in Preview but it means that your connector is not in general availability(GA). You can request to remove this “Preview” tag by moving your connector to GA<https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-to-ga>, if your connector meets our required criteria<https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-to-ga#required-criteria>. I tested your connector for GA and unfortunately the connector has 40% success rate. I would suggest to use the connector more and have more active connections for at least a month, and then we can test the connector again 😊.


1. Documentation link still shows secplugsscan
https://docs.microsoft.com/en-us/connectors/secplugscan/<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fconnectors%2Fsecplugscan%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Z%2BQ3luJNNUCky3fo6ol2FUrbdTaFJ%2B%2FbzBUEydmSZkg%3D&reserved=0>
If you are referring to the link, then this is something will not be changed with the connector name. the link will remain the same, but the docs will have the new name. please see below.

***@***.***

2. In the connector page https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fconnectors%2Fdetails%2Fshared_secplugscan%2Fscancloud%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lT0JUW69cDlpa2VUu%2BaE8BnIaYtMgf9UQDVEJDn0U7g%3D&reserved=0>
It is showing as secplugs connector - In the URL, connector name etc.,
The same thing will apply here for the link. The link will remain the same.

3. In the documentation "Getting started with your connector" section, there are some broken links and images, we have provided in our documentation.. It was there in our earlier secplugs documentation page and we want to remove that. It looks bad and needs to be removed.. We want to keep only one line in that section which is " Please follow ScanCloud Connector documentation here<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fscancloud.io%2Fdocs%3Fplugin%3Dplugin-secplugs-plugins-power-automate&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=FLNrRgAjuP4V60mmFU7Im6Zl7z9inYqBjxSY2rIeVS8%3D&reserved=0>".

Please remove the below section:
"
Creating a connection | DEFAULT | | | |---------------------------------------------|---------------------|---------------------------------| | Name | Type | Description | | API Key (from https://Scancloud.com/<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fscancloud.com%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=sK9B0jenHo1yyuGn3Lp1ZvRq65saFr77gEJh20dsGlQ%3D&reserved=0>) | securestring | The Api Key for this API |

Clean report Sample (For File): [Image removed by sender. enter image description here]

Malicious report Sample (For File): [Image removed by sender. enter image description here]
"
You can fix this in your Intro.md file (You can find an example of intro.md file on this page<https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio> at step 6).
For Images, please have them in an assets folder and you can just upload it to ISV using the .zip format. We don’t support images with link format in the docs.



Thanks
Shankar


On Tue, 9 Aug 2022 at 17:37, Shankar Jayaraman ***@***.******@***.***>> wrote:
Okay thanks for the update..  The below message we see in the activity center..  Hope this is the correct status..

"
Dear Partner,

Your connector is now onboarded to the current deployment schedule. You can expect another status update in 3-5 business days.

Best Regards, Srikanth Y.
"
Thanks
Shankar


On Tue, 9 Aug 2022 at 05:29, Amjed Ayoub (TEKsystems, Inc.) ***@***.******@***.***>> wrote:
Hello Shankar,

Thank you so much for reaching out.

You new update deployment did not start yet 😊. Hopefully, our engineers will prepare your update for Friday’s deployment train. Typically, it takes up to 3 to 4 weeks to complete.

Please let me know if you have any questions.

Thank you so much for your patience and understanding.

Best regards,
Amjed Ayoub

From: Shankar Jayaraman ***@***.******@***.***>>
Sent: Sunday, August 7, 2022 9:07 PM
To: microsoft/PowerPlatformConnectors ***@***.******@***.***>>
Cc: microsoft/PowerPlatformConnectors ***@***.******@***.***>>; Kartik Rao Polepalli ***@***.******@***.***>>; Connector Partner Mgmt Team ***@***.******@***.***>>; Srikanth Yenagandhula (MINDTREE LIMITED) ***@***.******@***.***>>
Subject: [EXTERNAL] Re: [microsoft/PowerPlatformConnectors] Rebranding of secplug to Scancloud APIs (PR #1668)

Powerplatform team,

I understand that scancloud connectors are now live in power automate connectors site.. I understand that you wanted to treat Scancloud.io connectors as new connectors not merging with existing secplugs connectors.. There seems to be some issue here..

1. Secplugs and scancloud connector documentation got mixed up..
https://docs.microsoft.com/en-us/connectors/secplugscan/<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fconnectors%2Fsecplugscan%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Z%2BQ3luJNNUCky3fo6ol2FUrbdTaFJ%2B%2FbzBUEydmSZkg%3D&reserved=0>
This is quite confusing - We would like to stick on to the latest documentation on scancloud connectors - you can remove the secplugs documentation all together if you want..

2. Scancloud connectors - says "scan newly added files using Secplugs"..
https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fconnectors%2Fdetails%2Fshared_secplugscan%2Fscancloud%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lT0JUW69cDlpa2VUu%2BaE8BnIaYtMgf9UQDVEJDn0U7g%3D&reserved=0>
As we mentioned earlier we want to replace secplugs completely with Scancloud.io - this is a rebranding exercise as mentioned earlier.. Please remove secplugs and replace them with Scancloud..  It should not get mixed up..

Please let us know if you have any questions.

thanks
Shankar

On Sat, 25 Jun 2022 at 02:41, v-EgorKozhadei ***@***.******@***.***>> wrote:

Hi @Ajaatsecplugs<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAjaatsecplugs&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670470570%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=2cBULDSYQ6UuaRqK0vDj5WV3QIk3rQY1iu9O9hat36M%3D&reserved=0>!

Your PR is approved and merged. Please submit an update in ISV Studio by using your latest commit ID.

Please let me know if you have any questions.

—
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FPowerPlatformConnectors%2Fpull%2F1668%23issuecomment-1165943366&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670626755%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lnOC151pPx2uVL7t49KiN1hV6q%2BprVQKZ%2FbqsxdI%2FqU%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOYRTYIFK4ZOTIML7DITG2TVQYP7ZANCNFSM5WTJFCMA&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670626755%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2BbybPfI1NOqd33wvrOa9IlrFLR8%2FEdsBTJq6tghB%2B7I%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

Ajaatsecplugs commented:
Thanks for the quick response..

We have now updated the intro.md in ISV studio - Could you please take the
updated intro.md file please..

Already regarding the connector failure, we were having some temporary
issues with our APIs and it was down, that could be reason for
the failure.. it is all fixed and it's been up and running for the last few
days without any issues. Could you please check now.

Thanks
Shankar

On Wed, 24 Aug 2022 at 10:30, Amjed Ayoub (TEKsystems, Inc.) <
***@***.***> wrote:

> Hello Shankar,
>
>
>
> Thank you so much for reaching out.
>
>
>
> Please see my answers inline below.
>
>
>
> Please let me know if you have any questions.
>
>
>
> Thank you so much again for your hard work with us.
>
>
>
> Best regards,
>
> Amjed Ayoub
>
>
>
>
>
>
>
>
>
> *From:* Shankar Jayaraman ***@***.***>
> *Sent:* Monday, August 22, 2022 6:14 AM
> *To:* Amjed Ayoub (TEKsystems, Inc.) ***@***.***>
> *Cc:* microsoft/PowerPlatformConnectors <
> ***@***.***>;
> microsoft/PowerPlatformConnectors <
> ***@***.***>; Kartik Rao Polepalli <
> ***@***.***>; Connector Partner Mgmt Team <
> ***@***.***>; Srikanth Yenagandhula
> (MINDTREE LIMITED) ***@***.***>
> *Subject:* Re: [EXTERNAL] Re: [microsoft/PowerPlatformConnectors]
> Rebranding of secplug to Scancloud APIs (PR #1668)
>
>
>
> Hi Team - There are some issues with scancloud connector preview  - I can
> see our scancloud rebranded connector updated as a preview
> https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fconnectors%2Fdetails%2Fshared_secplugscan%2Fscancloud%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lT0JUW69cDlpa2VUu%2BaE8BnIaYtMgf9UQDVEJDn0U7g%3D&reserved=0>
>  - Based on our earlier conversation, you have mentioned that you will be
> uploading this as a brand new scancloud connector, not an update to the
> existing secplugs connector. Also, we have the following concerns. Request
> to please address the same..
>
> All our new connectors will be published with this “Preview” tag. This
> does not mean that your connector is in Preview but it means that your
> connector is not in general availability(GA). You can request to remove
> this “Preview” tag by moving your connector to GA
> <https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-to-ga>,
> if your connector meets our required criteria
> <https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-to-ga#required-criteria>.
> I tested your connector for GA and unfortunately the connector has 40%
> success rate. I would suggest to use the connector more and have more
> active connections for at least a month, and then we can test the connector
> again 😊.
>
>
>
>
>
> 1. Documentation link still shows secplugsscan
>
> https://docs.microsoft.com/en-us/connectors/secplugscan/
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fconnectors%2Fsecplugscan%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Z%2BQ3luJNNUCky3fo6ol2FUrbdTaFJ%2B%2FbzBUEydmSZkg%3D&reserved=0>
>
> If you are referring to the link, then this is something will not be
> changed with the connector name. the link will remain the same, but the
> docs will have the new name. please see below.
>
>
>
>
>
> 2. In the connector page
> https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fconnectors%2Fdetails%2Fshared_secplugscan%2Fscancloud%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lT0JUW69cDlpa2VUu%2BaE8BnIaYtMgf9UQDVEJDn0U7g%3D&reserved=0>
>
> It is showing as secplugs connector - In the URL, connector name etc.,
>
> The same thing will apply here for the link. The link will remain the same.
>
>
>
> 3. In the documentation "Getting started with your connector" section,
> there are some broken links and images, we have provided in our
> documentation.. It was there in our earlier secplugs documentation page and
> we want to remove that. It looks bad and needs to be removed.. We want to
> keep only one line in that section which is " Please follow ScanCloud
> Connector documentation here
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fscancloud.io%2Fdocs%3Fplugin%3Dplugin-secplugs-plugins-power-automate&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=FLNrRgAjuP4V60mmFU7Im6Zl7z9inYqBjxSY2rIeVS8%3D&reserved=0>
> ".
>
>
>
> Please remove the below section:
>
> *"*
>
> *Creating a connection* | DEFAULT | | |
> |---------------------------------------------|---------------------|---------------------------------|
> | Name | Type | Description | | API Key (from https://Scancloud.com/
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fscancloud.com%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=sK9B0jenHo1yyuGn3Lp1ZvRq65saFr77gEJh20dsGlQ%3D&reserved=0>)
> | securestring | The Api Key for this API |
>
> *Clean report Sample (For File):* [image: Image removed by sender. enter
> image description here]
>
> *Malicious report Sample (For File):* [image: Image removed by sender.
> enter image description here]
>
> "
>
> You can fix this in your Intro.md file (You can find an example of
> intro.md file on this page
> <https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio>
> at step 6).
>
> For Images, please have them in an assets folder and you can just upload
> it to ISV using the .zip format. We don’t support images with link format
> in the docs.
>
>
>
>
>
>
>
> Thanks
>
> Shankar
>
>
>
>
>
> On Tue, 9 Aug 2022 at 17:37, Shankar Jayaraman ***@***.***>
> wrote:
>
> Okay thanks for the update..  The below message we see in the activity
> center..  Hope this is the correct status..
>
>
>
> "
>
> Dear Partner,
>
> Your connector is now onboarded to the current deployment schedule. You
> can expect another status update in 3-5 business days.
>
> Best Regards, Srikanth Y.
>
> "
>
> Thanks
>
> Shankar
>
>
>
>
>
> On Tue, 9 Aug 2022 at 05:29, Amjed Ayoub (TEKsystems, Inc.) <
> ***@***.***> wrote:
>
> Hello Shankar,
>
>
>
> Thank you so much for reaching out.
>
>
>
> You new update deployment did not start yet 😊. Hopefully, our engineers
> will prepare your update for Friday’s deployment train. Typically, it takes
> up to 3 to 4 weeks to complete.
>
>
>
> Please let me know if you have any questions.
>
>
>
> Thank you so much for your patience and understanding.
>
>
>
> Best regards,
>
> Amjed Ayoub
>
>
>
> *From:* Shankar Jayaraman ***@***.***>
> *Sent:* Sunday, August 7, 2022 9:07 PM
> *To:* microsoft/PowerPlatformConnectors <
> ***@***.***>
> *Cc:* microsoft/PowerPlatformConnectors <
> ***@***.***>; Kartik Rao Polepalli <
> ***@***.***>; Connector Partner Mgmt Team <
> ***@***.***>; Srikanth Yenagandhula
> (MINDTREE LIMITED) ***@***.***>
> *Subject:* [EXTERNAL] Re: [microsoft/PowerPlatformConnectors] Rebranding
> of secplug to Scancloud APIs (PR #1668)
>
>
>
> Powerplatform team,
>
>
>
> I understand that scancloud connectors are now live in power
> automate connectors site.. I understand that you wanted to treat
> Scancloud.io connectors as new connectors not merging with existing
> secplugs connectors.. There seems to be some issue here..
>
>
>
> 1. Secplugs and scancloud connector documentation got mixed up..
>
> https://docs.microsoft.com/en-us/connectors/secplugscan/
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fconnectors%2Fsecplugscan%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Z%2BQ3luJNNUCky3fo6ol2FUrbdTaFJ%2B%2FbzBUEydmSZkg%3D&reserved=0>
>
> This is quite confusing - We would like to stick on to the latest
> documentation on scancloud connectors - you can remove the secplugs
> documentation all together if you want..
>
>
>
> 2. Scancloud connectors - says "scan newly added files using Secplugs"..
>
>
> https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fconnectors%2Fdetails%2Fshared_secplugscan%2Fscancloud%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670001861%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lT0JUW69cDlpa2VUu%2BaE8BnIaYtMgf9UQDVEJDn0U7g%3D&reserved=0>
>
> As we mentioned earlier we want to replace secplugs completely with
> Scancloud.io - this is a rebranding exercise as mentioned earlier.. Please
> remove secplugs and replace them with Scancloud..  It should not get mixed
> up..
>
>
>
> Please let us know if you have any questions.
>
>
>
> thanks
>
> Shankar
>
>
>
> On Sat, 25 Jun 2022 at 02:41, v-EgorKozhadei ***@***.***>
> wrote:
>
> Hi @Ajaatsecplugs
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAjaatsecplugs&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670470570%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=2cBULDSYQ6UuaRqK0vDj5WV3QIk3rQY1iu9O9hat36M%3D&reserved=0>
> !
>
> Your PR is approved and merged. Please submit an update in ISV Studio by
> using your latest commit ID.
>
> Please let me know if you have any questions.
>
> —
> Reply to this email directly, view it on GitHub
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FPowerPlatformConnectors%2Fpull%2F1668%23issuecomment-1165943366&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670626755%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=lnOC151pPx2uVL7t49KiN1hV6q%2BprVQKZ%2FbqsxdI%2FqU%3D&reserved=0>,
> or unsubscribe
> <https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAOYRTYIFK4ZOTIML7DITG2TVQYP7ZANCNFSM5WTJFCMA&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7Cef4abcedc6a541abdb7008da84403b9b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637967708670626755%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2BbybPfI1NOqd33wvrOa9IlrFLR8%2FEdsBTJq6tghB%2B7I%3D&reserved=0>
> .
> You are receiving this because you were mentioned.[image: Image removed
> by sender.]Message ID: <microsoft/PowerPlatformConnectors/pull/1668/
> ***@***.***>
>
>
